### PR TITLE
changed 2015 to 2017 for VS build instructions

### PR DESCRIPTION
**Pull request by tomjebo**
We should at least add this change in the instructions. should we also add a note for the VS requirement now?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ This project uses the `project.json` for the project build system. Please see th
 
 To build the Open XML SDK:
 - Clone the repo at https://github.com/OfficeDev/Open-XML-SDK
-- Open the solution using Visual Studio 2015.  Community Edition of VS2015 works just fine.
+- Open the solution using Visual Studio 2017.  Community Edition of VS2017 works just fine.
```
Comment (twsouthwick) [inline README.md]: I would add that they will install side by side with previous versions of VS and provide a link

**Comment by tomjebo** [comment]
This addresses issue #189

File: README.md (hunk 1 above)
Comment (tomjebo) [inline README.md]: would this link be right: [https://msdn.microsoft.com/en-us/library/ms246609.aspx](url)?

Comment (twsouthwick) [inline README.md]: VS 2017 is actually not the only supported tool. I'd rephrase along the lines of "Open the solution with an editor that supports the latest .NET project files, such as VS 2017, VS Code, VS for Mac, or .NET CLI. See https://www.microsoft.com/net/download/core for details."

**Comment by twsouthwick** [comment]
fyi: if you change your initial post on this PR to have the line "Fixes #189", that issue will automatically close when this is merged.

**Comment by tomjebo** [comment]
thanks, that's what I was trying to do but thought it would close the issue when I commented. will do it next time. And I'll squash these two commits with the merge.

**Comment by twsouthwick** [comment]
You can edit the initial post to include it
